source/scss/clarendon.scss

```diff
@@ -0,0 +1,21 @@
+/*
+ * Clarendon
+ *
+ */
+@import 'shared';
+
+%clarendon,
+.clarendon {
+  @extend %filter-base;
+  filter: contrast(1.3) saturate(1.2);
```

Comment:
I'd bring up the saturating and bring down the contrast a bit: `contrast(1.2) saturate(1.35)`